src/debugger/apimDebug.ts

```diff
@@ -377,18 +378,15 @@ export class ApimDebugSession extends LoggingDebugSession {
 		}
 	}
 
-	private async getAccountCredentials(subscriptionId: string): Promise<TokenCredentialsBase> {
-		const azureAccountExtension = vscode.extensions.getExtension('ms-vscode.azure-account');
-		const azureAccount = azureAccountExtension!.exports;
-		await azureAccount.waitForFilters();
-		if (azureAccount.status !== 'LoggedIn') {
+	private async getAccountCredentials(): Promise<TokenCredential> {
+		const session = await AzureAuth.getReadySessionProvider();
+		if (GeneralUtils.failed(session)) {
```

Comment:
Is there any detail error message for this case?

Reply:
update the error message as "Failed to access the Azure Account Session."